pkg/cmd/cli/power/chassis.go

```diff
@@ -39,7 +43,16 @@ func NewChassisCommand() *cobra.Command {
 		Run: func(c *cobra.Command, args []string) {
 			// TODO: Translate power commands to available commands, since the available commands aren't very friendly.
```

Comment:
Think this TODO can go? Did you find consistency between the three vendors we commonly use?

Reply:
This is what I essentially did with the flags.  

Three commands (on, off, status) and then you use flag to modify their behavior (with `--force`, for example)